pkgs/development/tools/language-servers/fennel-ls/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchgit, lua, luaPackages }:
+stdenv.mkDerivation rec {
+  pname = "fennel-ls";
+  version = "0.1.0";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~xerool/fennel-ls/";
+    rev = version;
+    sha256 = "sha256-RW3WFJGwascD4YnnrAm/2LFnVigzgtfzVubLMDW9J5s=";
+  };
```

Consider using `finalAttrs` instead

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "fennel-ls";
  version = "0.1.0";

  src = fetchgit {
    url = "https://git.sr.ht/~xerool/fennel-ls/";
    rev = finalAttrs.version;
    sha256 = "sha256-RW3WFJGwascD4YnnrAm/2LFnVigzgtfzVubLMDW9J5s=";
  };
```

---

Thanks for the suggestions, what is the difference here ?